skabase/SKATestDevice/test/SKATestDevice_test.py

```diff
@@ -1,11 +1,12 @@
-#########################################################################################
+#!/usr/bin/env python
```

Comment:
Use new POGO to generate file.

Reply:
We will use this version now onwards.

Reply:
This file still needs to be corrected.  Basically keep original file (before PR), and add `test_On` and `test_Off`.

Reply:
Done.